File: Modul-1/Python/PRAK104-2410817210001-MuhammadFadillah.py
harga_A = 400000
harga_B = 350000
diskon_A = 0.13
diskon_B = 0.21

harga_diskonA = harga_A - (harga_A * diskon_A)
harga_diskonB = harga_B - (harga_B * diskon_B)

print(f"Harga sepatu A adalah {harga_A}")
print(f"Harga sepatu B adalah {harga_B}")
print(f"Sepatu A mendapat diskon 13% sehingga harganya menjadi {int(harga_diskonA)}")
print(f"Sepatu B mendapat diskon 21% sehingga harganya menjadi {int(harga_diskonB)}")

File: Modul-1/C/PRAK104-2410817210001-MuhammadFadillah.c
#include <stdio.h>

int main(){
    int harga_A = 400000;
    int harga_B = 350000;
    float diskon_A = 0.13;
    float diskon_B = 0.21;
    int harga_diskonA = harga_A - (harga_A * diskon_A);
    int harga_diskonB = harga_B - (harga_B * diskon_B);

    printf("Harga sepatu a adalah %d\n", harga_A); 
    printf("Harga sepatu b adalah %d\n", harga_B);
    printf("Sepatu A mendapatkan diskon 13%% sehingga harganya menjadi %d\n", harga_diskonA); 
    printf("Sepatu B mendapatkan diskon 21%% sehingga harganya menjadi %d\n", harga_diskonB);

    return 0; 
}